src/components/confirmation-dialogs.yml

```diff
@@ -16,3 +16,15 @@ deleteAllDescendantRevisions:
 restoreDocumentRevisionConfirmation:
   en: Do you really want to restore the content of revision {revisionId}?
   de: Möchten Sie den Inhalt der Revision {revisionId} wirklich wiederherstellen?
+confirmPassword:
+  en: Confirm password
+  de: Kennwortbestätigung
```

Comment:
Small thing, should it be 'Kennwort bestätigen' or 'Password confirmation', to match the phrasings between languages? 🤔

Reply:
I think it's a label for an input, so I would tend to use "Password confirmation" and "Kennwortbestätigung"